eZ/Bundle/EzPublishCoreBundle/EventListener/CacheViewResponseListener.php

```diff
@@ -66,7 +66,14 @@ public function configureCache(FilterResponseEvent $event)
         }
 
         $response->setPublic();
-        if ($this->enableTtlCache) {
+
+        if (($maxAge = $view->getMaxAge()) !== null) {
+            $response->setMaxAge($maxAge);
+        }
+
+        if (($sharedMaxAge = $view->getSharedMaxAge()) !== null) {
+            $response->setSharedMaxAge($sharedMaxAge);
+        } elseif ($this->enableTtlCache) {
```

Comment:
If one decides to specifically disable TTL cache, then it must not be activated by some custom code. Otherwise this setting is useless :wink: